modules/client/src/addressList/internal/client/estimation.ts

```diff
@@ -40,20 +39,20 @@ export class ClientAddressListEstimation extends ClientCore
     }
 
     const addresslistContract = AllowlistVoting__factory.connect(
-      _params.pluginAddress,
+      params.pluginAddress,
       signer,
     );
 
-    // TODO: Compute the cid instead of pinning the data on the cluster
-    let cid = "";
+    const startTimestamp = params.startDate?.getTime() || 0;
+    const endTimestamp = params.endDate?.getTime() || 0;
 
     const estimatedGasFee = await addresslistContract.estimateGas.createVote(
```

Comment:
Even if it will be changed later on, the estimation may not be accurate if the given value is `""`.  Let's not remove features that existed before, plz